docs/technical_reference/unit_models/zero_order_unit_models/anaerobic_digestion_oxidation_zo.rst

```diff
@@ -1,5 +1,5 @@
-Anaerobic Digestion (ZO)
-========================
+Anaerobic Digestion Oxidation (ZO)
```

Comment:
I thought I had added a title exception for this to leave out "Oxidation". Not sure how it came back, but can you check on this and see if we could get it back to omitting "Oxidation"?

Reply:
Oh, yea I changed the name in the Excel file to explicitly distinguish it from Anaerobic Digestion Reactive. But I can remove the oxidation again if it's not necessary to distinguish the two.